mod-data-export/mod-data-export.postman_collection.json

```diff
@@ -1,1791 +1,1865 @@
-      "protocolProfileBehavior": {}
-    }
-  ],
-  "event": [
-    {
-      "listen": "prerequest",
-      "script": {
-        "id": "fef69b0c-ae3b-4683-adc3-e3cb7031d157",
-        "type": "text/javascript",
-        "exec": [
-          "let testData = {",
-          "    // User template with hardcoded id",
-          "    users: {",
-          "        dikuadmin : {",
-          "            credentials: {",
-          "                \"username\": \"diku_admin\",",
-          "                \"password\": \"admin\"",
-          "            }",
-          "        },",
-          "        admin: {",
-          "            user: {",
-          "                \"id\": \"00000000-1111-5555-9999-999999999999\",",
-          "                \"username\": \"mdexp-admin-user\",",
-          "                \"active\": true,",
-          "                \"personal\": {",
-          "                    \"firstName\": \"Test\",",
-          "                    \"lastName\": \"Api\"",
-          "                }",
-          "            },",
-          "            credentials: {",
-          "                \"username\": \"mdexp-admin-user\",",
-          "                \"password\": \"mdexp-admin-password\"",
-          "            },",
-          "            permissions: {",
-          "                \"userId\": \"00000000-1111-5555-9999-999999999999\",",
-          "                \"permissions\": []",
-          "            }",
-          "        },",
-          "        regular: {",
-          "            user: {",
-          "                \"id\": \"00000001-1111-5555-9999-999999999999\",",
-          "                \"username\": \"mdexp-user\",",
-          "                \"active\": true,",
-          "                \"personal\": {",
-          "                    \"firstName\": \"Regular\",",
-          "                    \"lastName\": \"API Tests\"",
-          "                }",
-          "            },",
-          "            credentials: {",
-          "                \"username\": \"mdexp-user\",",
-          "                \"password\": \"mdexp-password\"",
-          "            },",
-          "            permissions: {",
-          "                \"userId\": \"00000001-1111-5555-9999-999999999999\",",
-          "                \"permissions\": [",
-          "                    \"data-export.export.post\",",
-          "                    \"data-export.file-definitions.item.get\",",
-          "                    \"data-export.file-definitions.item.post\",",
-          "                    \"data-export.file-definitions.upload.post\",",
-          "                    \"data-export.job-executions.items.get\"",
-          "                ]",
-          "            }",
-          "        },",
-          "    },",
-          "    tenant: {",
-          "        \"id\": pm.variables.get(\"testTenant\"),",
-          "        \"name\": \"Test mod-data-export tenant\",",
-       
```

Comment:
I wonder why the entire file is showing as different. Were there any major changeS?

Reply:
I don't know what exactly reason, I used export option in postman to update the collection, I will investigate it and try to fix

Reply:
Whitespaces ...

Reply:
I kept formatting with tabs instead of whitespaces as it's standard postman formatting and the same used for all other collections in folio-api-tests. But pushed two commits, first one contains changes of formatting, the second one contains changes relate to MDEXP-74 directly. Therefore you can check the second commit to see real changes.